### ISSUE TITLE
Formedit giving error while any event occurs in ui   

### ISSUE THREAD

**Issue by raikhanworkspace013** [formio/react#580]
Failed to execute 'structuredClone' on 'Window': options => {
            this.submit(false, options).catch(e => {
                options.instance.loading = fa...<omitted>... } could not be cloned.
Error: Failed to execute 'structuredClone' on 'Window': options => {
            this.submit(false, options).catch(e => {
                options.instance.loading = fa...<omitted>... } could not be cloned.
    at onAnyEvent (webpack://oss-portal/./node_modules/@formio/react/lib/components/Form.js?:36:24)
    at EventEmitter.eval (webpack://oss-portal/./node_modules/@formio/react/lib/components/Form.js?:230:53)
    at EventEmitter.emit (webpack://oss-portal/./node_modules/eventemitter3/index.js?:182:35)
    at EventEmitter.emit (webpack://oss-portal/./node_modules/@formio/js/lib/cjs/EventEmitter.js?:47:19)
    at EmailComponent.emit (webpack://oss-portal/./node_modules/@formio/js/lib/cjs/Element.js?:180:25)
    at HTMLInputElement.eval (webpack://oss-portal/./node_modules/@formio/js/lib/cjs/components/_classes/component/Component.js?:2877:22)

**Comment by lane-formio** [comment]
Version?

**Comment by raikhanworkspace013** [comment]
> Version?

      "@formio/js": "5.0.0-dev.5621.91bd945",
        "@formio/react": "6.0.0-dev.568.ced0fbd",

**Comment by raikhanworkspace013** [comment]
@lane-formio one more issue i am facing when I use formedit component and create a new form setting values not set in payload

**Comment by lane-formio** [comment]
<div>Agent Olga Bannikova linked Freshdesk ticket <a href="https://formhelp.freshdesk.com/a/tickets/49512">49512</a> for this issue.</div><br/>

**Comment by lane-formio** [comment]
I have this coming up in the queue for review, in the meantime, could you provide a codesandbox as an example? 

Also, I published @formio/react@6.0.0-rc.1 if you can update to that and try the latest @formio/js rc version.

**Comment by piotr-huk** [comment]
> I have this coming up in the queue for review, in the meantime, could you provide a codesandbox as an example?
> 
> Also, I published @formio/react@6.0.0-rc.1 if you can update to that and try the latest @formio/js rc version.

I had the same problem - the exception mentioned in the first post causes the inability to save the form.  Updating the packages helped. Here are the versions that work correctly:

```        
  "@formio/js": "5.0.0-rc.76",
  "@formio/react": "6.0.0-rc.1",
```

**Comment by lane-formio** [comment]
Thanks @piotr-huk
I'm going to go ahead close the issue as resolved.
